docs/HyperIndex/Guides/event-handlers.mdx

```diff
@@ -18,78 +18,43 @@ envio codegen
 
 in the project directory to generate the functions you will use in your handlers.
 
-Each event requires two functions to be registered:
+Each event requires that a handler be registerred. These functions are asyncronous.
```

Comment:
I don't think we require to register an event. It's just not processed when a handler is not registered.

Reply:
Hmm, yeah, you're right. I'll re-word slightly.

```suggestion
Each event that you want to process requires that a handler be registered. These functions are asynchronous.
```